transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -599,6 +599,16 @@ internal class MessagingClientImpl(
     }
 }
 
+/**
+ * Checks if the string contains both the words "conversation" and "clear" (case-insensitive).
+ *
+ * @return `true` if the string contains both "conversation" and "clear", `false` otherwise.
+ */
+private fun String?.isClearConversationError(): Boolean {
+    val regex = Regex("(?=.*\\bconversation\\b)(?=.*\\bclear\\b)", RegexOption.IGNORE_CASE)
```

Comment:
I think you need to set the regexp in a way that keeps the `conversation` `clear` words together without other characters between (maybe only spaces) other wise it may take other errors that mentions those 2 words